<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sepaCreditTransferEdit</class>
 <widget class="QWidget" name="sepaCreditTransferEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="2">
    <widget class="KLed" name="statusIban">
     <property name="state">
      <enum>KLed::On</enum>
     </property>
     <property name="color">
      <color>
       <red>255</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="KLineEdit" name="beneficiaryAccNum"/>
   </item>
   <item row="6" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Amount</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="kMyMoneyEdit" name="value">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="resetButtonVisibility" stdset="0">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="1">
    <widget class="KLineEdit" name="sepaReference"/>
   </item>
   <item row="7" column="2">
    <widget class="KLed" name="statusReference">
     <property name="state">
      <enum>KLed::On</enum>
     </property>
     <property name="color">
      <color>
       <red>255</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Purpose</string>
     </property>
     <property name="buddy">
      <cstring>purpose</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="KLed" name="statusAmount">
     <property name="state">
      <enum>KLed::On</enum>
     </property>
     <property name="color">
      <color>
       <red>255</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_iban">
     <property name="text">
      <string>IBAN</string>
     </property>
     <property name="buddy">
      <cstring>beneficiaryAccNum</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="KLed" name="statusBic">
     <property name="state">
      <enum>KLed::On</enum>
     </property>
     <property name="color">
      <color>
       <red>255</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="KLineEdit" name="beneficiaryBankCode"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>End-to-end reference</string>
     </property>
     <property name="buddy">
      <cstring>sepaReference</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="KLed" name="statusPurpose">
     <property name="state">
      <enum>KLed::On</enum>
     </property>
     <property name="color">
      <color>
       <red>255</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="3">
    <widget class="KMyMoneyTextEdit" name="purpose">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>30</height>
      </size>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::FixedColumnWidth</enum>
     </property>
     <property name="lineWrapColumnOrWidth">
      <number>27</number>
     </property>
     <property name="acceptRichText">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="KLineEdit" name="beneficiaryName"/>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Beneficiary Name</string>
     </property>
     <property name="buddy">
      <cstring>beneficiaryName</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="KLed" name="statusBeneficiaryName">
     <property name="state">
      <enum>KLed::On</enum>
     </property>
     <property name="color">
      <color>
       <red>255</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_bic">
     <property name="text">
      <string>BIC</string>
     </property>
     <property name="buddy">
      <cstring>beneficiaryBankCode</cstring>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>label</zorder>
  <zorder>sepaReference</zorder>
  <zorder>purpose</zorder>
  <zorder>label_4</zorder>
  <zorder>label_2</zorder>
  <zorder>beneficiaryName</zorder>
  <zorder>beneficiaryAccNum</zorder>
  <zorder>label_bic</zorder>
  <zorder>label_iban</zorder>
  <zorder>beneficiaryBankCode</zorder>
  <zorder>statusPurpose</zorder>
  <zorder>statusReference</zorder>
  <zorder>statusAmount</zorder>
  <zorder>statusBic</zorder>
  <zorder>statusIban</zorder>
  <zorder>statusBeneficiaryName</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>kMyMoneyEdit</class>
   <extends>QFrame</extends>
   <header>kmymoneyedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KLed</class>
   <extends>QWidget</extends>
   <header>kled.h</header>
  </customwidget>
  <customwidget>
   <class>KMyMoneyTextEdit</class>
   <extends>QTextEdit</extends>
   <header>kmymoneytextedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>beneficiaryName</tabstop>
  <tabstop>beneficiaryAccNum</tabstop>
  <tabstop>beneficiaryBankCode</tabstop>
  <tabstop>sepaReference</tabstop>
  <tabstop>purpose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
